src/FlightMap/Widgets/QGCInstrumentWidget.qml

```diff
@@ -55,6 +59,111 @@ ColumnLayout {
             vehicle:                globals.activeVehicle
             anchors.verticalCenter: parent.verticalCenter
         }
+    }  
+
+    enum WinchCommands {
+        RELEASE = 0,
+        DELIVER = 4,
+        RETRACT = 6
```

Comment:
Intuitively I'd say that RELEASE sounds like WINCH_ABANDON_LINE, but here it seems that RELEASE maps to WINCH_RELAXED. Since this enum is basically an "import" of the MAVLINK commands, I suggest naming them according to the defitinition.

On that note: Do you really need to re-define these variables? Aren't the WINCH_ACTION messages included in the Mavlink library?

Reply:
The MAVLink library is not updated with all winch commands, only a couple are included for some reason. In addition, none of the winch commands are included in the QML MAVLink enum. Added comments on where I got the numbers from instead, which follows the (not yet implemented) MAVLink standard.